babel.config.js

```diff
@@ -10,38 +10,45 @@ const TARGETS = {
 const CONFIG = {
   default: {
     presets: [
-      ['@babel/env', {
-        targets: TARGETS
-      }]
+      // Rewritten by each target
     ],
     plugins: [
-      ['babel-plugin-inline-import', {
-        extensions: [
-          '.worker.js'
-        ]
-      }]
     ],
     ignore: [
       '**/*.worker.js'
     ]
   }
 };
 
+const PLUGINS = [
+  ['babel-plugin-inline-import', {
+    extensions: [
+      '.worker.js'
+    ]
+  }]
+];
+
 CONFIG.es6 = Object.assign({}, CONFIG.default, {
   presets: [
     ['@babel/env', {
       targets: TARGETS,
       modules: false
     }]
-  ]
+  ],
+  plugins: PLUGINS.concat([
+    ['@babel/plugin-transform-runtime', {useESModules: true}]
```

Comment:
My understanding is that babel runtime is used to replace polyfill. The `es6` targets surely do not need it?

Reply:
> My understanding is that babel runtime is used to replace polyfill. The `es6` targets surely do not need it?

The ES6 targets use babel-preset-env with a selection of browser and node versions. That removes all babel transforms that are supported by those versions, but still includes a few transforms to support e.g. Edge, Node 8, Safari. It is possible to set a verbose flag on preset-env to make it print the selected list.

In particular, now that we start using ES2018 syntax we will hit more of these (preset-env handles all transforms that have reached stage4, so it is presumably ES2018+growing subset of ES2019).